src/Compilers/CSharp/Portable/Syntax/SyntaxKindFacts.cs

```diff
@@ -143,7 +143,7 @@ public static IEnumerable<SyntaxKind> GetPreprocessorKeywordKinds()
 
         public static bool IsPunctuation(SyntaxKind kind)
         {
-            return kind >= SyntaxKind.TildeToken && kind <= SyntaxKind.QuestionQuestionEqualsToken;
+            return kind >= SyntaxKind.TildeToken && kind <= SyntaxKind.GreaterThanGreaterThanGreaterThanEqualsToken;
```

Comment:
This is also including !! that wasn't included before; that seems reasonable to me but we think that was just an oversight before?

Reply:
> This is also including !! that wasn't included before; that seems reasonable to me but we think that was just an oversight before?

I think it was. CC @RikkiGibson

Reply:
Yes, it was an oversight. It looks like this could change behavior of classification of `!!`, but I must not have added any tests for `!!` to `src/EditorFeatures/CSharpTest/Classification/SyntacticClassifierTests.cs`.